logstash-mixin-config_deprecation_support.gemspec

```
@@ -0,0 +1,23 @@
+Gem::Specification.new do |s|
+  s.name          = 'logstash-mixin-config_deprecation_support'
+  s.version       = "1.0.0"
+  s.licenses      = %w(Apache-2.0)
+  s.summary       = "Support for Logstash plugins wishing to deprecate config options"
+  s.description   = "This gem is meant to be a dependency of any Logstash plugin that wishes to deprecate options."
+  s.authors       = %w(Elastic)
+  s.email         = 'info@elastic.co'
+  s.homepage      = 'https://github.com/logstash-plugins/logstash-mixin-config_deprecation_support'
+  s.require_paths = %w(lib vendor/jar-dependencies)
```

Nit: no need to add `vendor/jar_dependencies` to the require path (problem likely also exists in the upstream example I pointed you to 🤦🏼‍♂️ )

```suggestion
  s.require_paths = %w(lib)
```